app/views/layouts/viewer_layout.html.erb

```diff
@@ -3,13 +3,14 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
     <link rel="stylesheet" type="text/css" href="uv/uv.css">
     <%= stylesheet_link_tag 'viewer' %>
-    <script type="text/javascript" src="uv/lib/offline.js"></script>
-    <script type="text/javascript" src="uv/lib/offline.js"></script>
-    <script type="text/javascript" src="uv/helpers.js"></script>
+    <%= yield :head %>
+    <script type="text/javascript" src="/uv/lib/offline.js"></script>
+    <script type="text/javascript" src="/uv/lib/offline.js"></script>
+    <script type="text/javascript" src="/uv/helpers.js"></script>
   </head>
   <body>
     <%= yield %>
     <%= javascript_include_tag 'viewer' %>
-    <script type="text/javascript" src="uv/uv.js"></script>
+    <script type="text/javascript" src="/uv/uv.js"></script>
```

Comment:
Was this previously broken? What bug was observed?

Reply:
Yeah - when it was on auth you'd get 404s for these, since it was trying to load '/auth/uv/uv.js'